common/app/rendering/core/JavascriptRendering.scala

```diff
@@ -6,13 +6,49 @@ import java.nio.file.{Files, Paths}
 import javax.script.{CompiledScript, SimpleScriptContext}
 
 import common.Logging
-import play.api.libs.json.{JsValue, Json, JsObject}
+import play.api.libs.json._
 import rendering.core.JavascriptEngine.EvalResult
 
 import scala.util.{Failure, Try}
 
 trait JavascriptRendering extends Logging {
 
+  case class TypeFace(typeFace: String, fileTypes: List[FileType])
+
+  object TypeFace {
+    implicit val typeFaceWriter: Writes[TypeFace] = Json.writes[TypeFace]
+  }
+
+  case class FileType(fileType: String, endpoint: String, hintTypes: List[HintType])
+
+  object FileType {
+    implicit val typeFaceWriter: Writes[FileType] = Json.writes[FileType]
```

Comment:
Should this value be called `fileTypeWriter`? Same question applies to the `HintType` companion object